docs/pages/dsHDMIIn/ds-Hdmi-In_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,216 @@
+# Device Settings HdmiIn L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings HdmiIn module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
```

Comment:
EDID, ALLM,  AVI, SPD, HdmiIn requires expand details

Reply:
The relevant usage in the document should be in `ALLM` or `AVI` format. Please follow in that way